web/client/components/style/ThemaClassesEditor.jsx

```diff
@@ -40,16 +39,13 @@ class ThemaClassesEditor extends React.Component {
                 </Col>
                 <Col xs="4">
                     <NumberPicker
-                        format="- ###.###"
                         value={classItem.min}
                         onChange={(value) => this.updateMin(index, value)}
                     />
                 </Col>
                 <Col xs="4">
                     <NumberPicker
-                        format="- ###.###"
                         value={classItem.max}
-                        precision={3}
```

Comment:
So, we also lost the ability to specify precision, but I can see precision handling in the component. How can I use it?

Reply:
I think I removed this because is misaligned with min  NumberPicker, so I thought it was a typo.